src/index.html

```diff
@@ -640,7 +640,7 @@ <h5>Acknowledgement</h5>
 
                                             <b>Is your data represented in a valid VCF file?</b>
                                             <p>
-                                                Genetic variants submitted to the EVA must be described in the <a href="http://samtools.github.io/hts-specs/" target="_blank">Variant Call Format (VCF)</a>. The EVA has developed a custom <a href="https://github.com/ebivariation/vcf-validator" target="_blank">VCF file validator</a> and accepts submission of only VCF files that adhere to the specification. Furthermore, VCF files submitted to the EVA should provide either sample genotypes and/or aggregated sample summary-level allele frequencies.
+                                                Genetic variants submitted to the EVA must be described in the <a href="http://samtools.github.io/hts-specs/" target="_blank">Variant Call Format (VCF)</a>. The EVA has developed the <a href="https://github.com/ebivariation/vcf-validator" target="_blank">EVA VCF validation suite</a> and accepts submission of only VCF files that adhere to the specification and match a reference sequence. Furthermore, VCF files submitted to the EVA should provide either sample genotypes and/or aggregated sample summary-level allele frequencies.

```

Comment:
The second sentence sounds as if the VCF validation suite accepts only submission of valid VCF files. Can we re-phrase to "EVA only accepts VCF files that adhere to the specification and match a reference sequence. Submitters can make use of the EVA VCF validation suite to ensure such conformance. Furthermore..."